packages/swagger-ui/README.md

```diff
@@ -0,0 +1,81 @@
+# Swagger UI Middleware and Component for Hono
+
+This library, `@hono/swagger-ui`, provides a middleware and a component for integrating Swagger UI with Hono applications. Swagger UI is an interactive documentation interface for RESTful APIs, making it easier to understand and test API endpoints.
```

Comment:
I think it would be better to specify that the API is compliant with the [OpenAPI Spec](https://swagger.io/specification/), not "RESTful APIs."